### PR TITLE
Reuse existing domain and consolidate monitoring documentation

### PR DESCRIPTION
**Pull request by gortegaMS**
### [Issue-754: K8s version downgraded to v1.19.9 ](https://github.com/retaildevcrews/ngsa/issues/754)

Closes
  https://github.com/retaildevcrews/ngsa/issues/732
  https://github.com/retaildevcrews/ngsa/issues/758
  https://github.com/retaildevcrews/ngsa/issues/754

## PR Checklist

---

- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [X] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Description

- Terraform scripts updated to reuse and deploy to an existing domain and dns zone
- Monitoring Traefik Ingress and PnP documentation consolidated 
- K8s version downgraded to v1.19.9

 

## Does this introduce a breaking change

- [ ] YES
- [X] NO

## Testing

Please follow instruction given in files "01-terraform.md" and "02-aks.md" from branch "gortegaMS-732"

### CODE REVIEW

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md [hunk 1]
```diff
@@ -97,8 +97,39 @@ az account set -s {subscription name or id}
...
+```
+
+### Domain name and alias reord configuration
+
+```bash
+
+The following instructions assume that there is an existing App Service Domain registerd and its corresponding DNS Zone has been configured.
+
+# Navigate to application gateway directory 
+
```
Comment (kev-ms) [inline enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md]: NIT: extra lines between the instructions and the commands per rest of the repo

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md [hunk 2]
```diff
@@ -97,8 +97,39 @@ az account set -s {subscription name or id}
 # If you are running in Azure Cloud Shell, you need to run the following additional command:
 export TF_VAR_logged_user_objectId=$(az ad signed-in-user show --query objectId -o tsv)
 
+
```
Comment (kev-ms) [inline enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md]: Move this section `Terraform setup & environment deployment` after `Domain name and alias reord configuration
`? Currently deployment steps are pushed down but this section is still here

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/02-aks.md [hunk 3]
```diff
@@ -133,7 +137,6 @@ Please review the Baseline components that are deployed at [cluster-baseline-set
     kubectl get ingress -n a0008
     # This website will be available at the public domain below
```
Comment (kev-ms) [inline enterprise_scale/construction_sets/aks/online/aks_secure_baseline/02-aks.md:138]: Remove this comment or list sample domain with application context root?
examples:
`https://<subdomain>.<domain>/memory`
` https://<subdomain>.<domain>/loderunner`

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/configuration/agw/agw_application.tfvars [hunk 4]
```diff
@@ -9,12 +9,7 @@ application_gateway_applications = {
         name                           = "public-443"
         front_end_ip_configuration_key = "public"
         front_end_port_key             = "443"
-        # host_name                      = "www.y4plq60ubbbiop9w1dh36tlgfpxqctfj.com"
-        dns_zone = {
-          key = "dns_zone1"
-          record_type = "a"
-          record_key = "agw"
-        }
```
Comment (kev-ms) [inline enterprise_scale/construction_sets/aks/online/aks_secure_baseline/configuration/agw/agw_application.tfvars:17]: Probably comment this out and keep there for reference?

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/traefik/README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+            kind: ClusterRole
+            name: traefik-ingress-controller
+        subjects:
+        - kind: ServiceAccount
+            name: traefik-ingress-controller
+            namespace: a0008 # This is the svc account ns, where traefik is
+    EOF
+    ```
+
+- Apply your app's yaml file
```
Comment (kev-ms) [inline enterprise_scale/construction_sets/aks/online/aks_secure_baseline/traefik/README.md:148]: Missing associated command?

**Review by kev-ms — CHANGES_REQUESTED** [review]

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md (hunk 1 above)
Comment (gortegaMS) [reply enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md]: fixed

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md (hunk 2 above)
Comment (gortegaMS) [reply enterprise_scale/construction_sets/aks/online/aks_secure_baseline/01-terraform.md]: fixed

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/02-aks.md (hunk 3 above)
Comment (gortegaMS) [reply enterprise_scale/construction_sets/aks/online/aks_secure_baseline/02-aks.md:138]: fixed

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/traefik/README.md (hunk 5 above)
Comment (gortegaMS) [reply enterprise_scale/construction_sets/aks/online/aks_secure_baseline/traefik/README.md:148]: fixed

File: enterprise_scale/construction_sets/aks/online/aks_secure_baseline/configuration/agw/agw_application.tfvars (hunk 4 above)
Comment (gortegaMS) [reply enterprise_scale/construction_sets/aks/online/aks_secure_baseline/configuration/agw/agw_application.tfvars:17]: I think it is not a good practice to leave code commented out that is not fully functionally, besides, if we want to keep it for reference, we have three sources to refer to 
github history , the original CAF project and azure caf project examples.

**Review by kev-ms — APPROVED** [review]
Looks good. Just need an adjustment on title for domain name per our conversation

**Pull request merged**